testing/amp-performance-extension/measure.js

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function renderMeasurement(container, label, count) {
```

Comment:
Nit: `innerHTML`, since we control the `label` and `count`?

Reply:
I return the countSpan here so if we use innerHTML there will be an extra querySelector so pretty much the same amount of work... Leaving it for now.

Reply:
I disagree, the work is considerably simpler (and much easier to reason about):

```js
container.innerHTML = `<div class='i-amphtml-performance-line'>
  <div class="i-amphtml-performance-label">${label}</div>
  <div class="i-amphtml-performance-count">${count.toFixed(4)}</div>
</div>`;
return container.lastElementChild
```